src/SprykerEco/Shared/Payone/PayoneConstants.php

```diff
@@ -48,4 +48,8 @@
     const GLOSSARY_FILE_PATH = 'Business/Internal/glossary.yml';
 
     const HOST_YVES = 'HOST_YVES';
+
+    const PAYONE_BANCONTACT_AVAILABLE_COUNTRIES = [
```

Comment:
This is the config constant and not the environment one.
The proper place is Shared/Payone/PayoutConfig.php